src/main/java/mekanism/common/base/EnergyAcceptorWrapper.java

```diff
@@ -140,6 +140,10 @@ public IC2Acceptor(IEnergySink ic2Acceptor) {
         @Override
         public double acceptEnergy(EnumFacing side, double amount, boolean simulate) {
             double toTransfer = Math.min(Math.min(acceptor.getDemandedEnergy(), toEU(amount)), Integer.MAX_VALUE);
```

Comment:
not your code, but should we remove the MAX_VALUE cap there? since EU appears to be a `double` now

Reply:
Done